docs/python-grader/Makefile

```diff
@@ -0,0 +1,12 @@
+# This should be run inside the centos-plbase container
```

Comment:
Please add this as a recipe in the [`docs/Makefile`](https://github.com/PrairieLearn/PrairieLearn/blob/6bbca3f93e24bfcbb12b34637cc17b5983644d0e/docs/Makefile).